### ISSUE TITLE
Use install.sh

### ISSUE THREAD

**Issue by lamont-granquist** [opscode-cookbooks/chef-server#53]
If this cookbook switched to downloading install.sh and then executing it, it would be able to leverage updates to install.sh which would avoid fixing issues in two places.

This cookbook and the omnibus-updater cookbook both suffer from the problem of having bad, old buggy pure-ruby implementations of install.sh which encounter problems that are solved in install.sh, but which need to get ported.

If there's additional flags and checks that install.sh needs to do for idempotency, then those requirements should get pushed into install.sh

**Comment by jtimberman** [comment]
This was opened long before we started publishing Chef Server 12 packages to Package Cloud. That is a far saner way to get the packages delivered via a cookbook than using omnitruck scripts. I'm going to close this as won't fix, as any refactoring of this cookbook will be for a Chef Server 12 world.
